sopa/segmentation/stainings.py

```diff
@@ -31,6 +31,7 @@ def __init__(
         image_key: str | None = None,
         min_area: float = 0,
         clip_limit: float = 0.2,
+        clahe_kernel_size: Optional[Union[int, Iterable[int]]] = None,
```

Comment:
Can you add docstrings for this parameter?

Reply:
And can you use the python>=3.10 typing syntax? I.e., `int | Iterable[int] | None`